src/core/ext/filters/backend_metrics/backend_metric_filter.h

```diff
@@ -35,8 +35,8 @@ class BackendMetricFilter : public ImplementChannelFilter<BackendMetricFilter> {
  public:
   static const grpc_channel_filter kFilter;
 
-  static absl::StatusOr<BackendMetricFilter> Create(const ChannelArgs& args,
-                                                    ChannelFilter::Args);
+  static absl::StatusOr<std::unique_ptr<BackendMetricFilter>> Create(
+      const ChannelArgs& args, ChannelFilter::Args = {});
```

Comment:
Why the default for the second param?  Doesn't seem like it would ever be called without that arg?

Looks like this same change was made for a number of filters.

Reply:
I was experimenting with eliminating it, but I don't think that's the right path now. Looks like it slipped in though, cleaning up.